samples/Azure.Management.Storage/Generated/BlobContainerContainer.cs

```diff
@@ -211,106 +210,6 @@ public async override Task<Response<BlobContainer>> GetAsync(string containerNam
             }
         }
 
-        /// <summary> Filters the list of <see cref="BlobContainer" /> for this resource group. </summary>
-        /// <param name="top"> The number of results to return. </param>
-        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="P:System.Threading.CancellationToken.None" />. </param>
-        /// <returns> A collection of <see cref="BlobContainer" /> that may take multiple service requests to iterate over. </returns>
-        public Pageable<BlobContainer> List(int? top = null, CancellationToken cancellationToken = default)
```

Comment:
this one should not be removed, only the namefilter one which ends up calling the extra Get method on line 261